### ISSUE TITLE
[Accessibility] Radio Button Group: "Clear" button doesn't have the `aria-label` attribute

### ISSUE THREAD

**Issue by andrewtelnov** [surveyjs/survey-library#10795]
aria-label attribute is missed for "Clear" button in radio group question.
The issue reported is [here](https://surveyjs.answerdesk.io/internal/ticket/details/T26024),

**Comment by JaneSjs** [comment]
@dmitry-kurmanov, it appears that when a user navigates to a RadioGroup question using the Tab key, focus lands on the Clear button. At this point, a screen reader does not announce any contextual information about the question to which this button belongs.

Consider the following demo:
```
{
  "pages": [
    {
      "name": "page1",
      "elements": [
        {
          "type": "text",
          "name": "question2"
        },
        {
          "type": "radiogroup",
          "name": "question1",
          "title": "SELECT YOUR PREFFERED ITEM",
          "defaultValue": "Item 2",
          "choices": [
            "Item 1",
            "Item 2",
            "Item 3"
          ],
          "allowClear": true
        }
      ]
    }
  ]
}
```

<img width="1544" height="1000" alt="Image" src="https://github.com/user-attachments/assets/9473e3c4-9855-4ffb-8f4d-16331e8aa49e" />

When the Clear button receives focus, the screen reader announces only the button itself, without indicating which question’s value will be cleared.

Please check whether the question title (or some other contextual information) should be announced when the Clear button within a question receives focus for the first time.
